src/test/java/com/futurecollars/lesson7/zad5/RectangularTriangleTest.java

```diff
@@ -0,0 +1,20 @@
+package com.futurecollars.lesson7.zad5;
+
+import com.futurcollars.lesson7.zad5.RectangularTriangle;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class RectangularTriangleTest {
+
+    @Test
+    void shouldCheckRectangularTriangle() {
+        // given
+        int sideA = 6;
+        int sideB = 8;
+        int sideC = 10;
+        // when
+        boolean resultTriangle = RectangularTriangle.getCheckRectangularTriangle(sideA, sideB, sideC);
```

Comment:
ponownie wystarczy result, niepotrzebnie kombinujesz i te nazwy zmiennych są średnie wtedy, a tak masz prostą zasadę, to co w when to nazywasz result :)